plugins/consul/consul.go

```diff
@@ -184,7 +184,11 @@ func Cmd(updaterFlags rotor.UpdaterFromFlags) *command.Cmd {
 		&runner.tbnServiceTag,
 		"cluster-tag",
 		defaultClusterTag,
-		"The tag used to indicate that a service should be imported as a Cluster.")
+		`
+			The tag used to indicate that a service should be imported as a Cluster.
+			If used in conjunction with 'tag-delimiter' its value can be used as 
+			a public URL i.e. tbn-cluster:example.com"
```

Comment:
"its value can be used to override the cluster name from the default value of the name of the service in consul." 

also, we'd prefer to keep one long string in double quotes.